lib/data/user.dart

```diff
@@ -17,9 +18,12 @@ class UserProfile {
       username: json["username"],
       profileImage: json["imageUrl"],
       pronouns: json["pronouns"],
+      points: 100,
```

Comment:
Why starting with 100?
We do not need this. All the users will start with 0 points.

Reply:
First data will come from the phone hahaha.